deploy/bundle.yaml

```diff
@@ -759,14 +760,21 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: percona-xtradb-cluster-operator
-        image: perconalab/percona-xtradb-cluster-operator:main
+        image: percona/percona-xtradb-cluster-operator:1.10.0
```

Comment:
but we need to use main tag

Reply:
Correct, but I will make another PR with changes needed after release. It was done for PSMDB the same: https://github.com/percona/percona-server-mongodb-operator/commit/9010c36fa1ca3f50069cb3240bc449e7119496a6